FightPandemics/Features/Feed/Card/FeedBodyTxt.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  FeedBodyTxt.swift
+//  FightPandemics
+//
+//  Created by ehsan sat on 5/26/20.
+//
+//  Copyright (c) 2020 FightPandemics
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+import UIKit
+
+class FeedBodyTxt: UIView {
+    private var bodyLbl = UILabel()
+    private var viewMore = FeedViewMoreBtn()
+    var text: String
+    init(text: String) {
+        self.text = text
+        super.init(frame: .zero)
+        setUp()
+    }
+
+    required init?(coder _: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setUp() {
+        let newTxt = text.components(separatedBy: " ").dropLast().dropLast().dropLast().joined(separator: " ")
+        bodyLbl.attributedText = NSAttributedString(string: newTxt, attributes: [NSAttributedString.Key.font: Fonts.poppinsRegular.customFont(size: 14), NSAttributedString.Key.foregroundColor: UIColor.fightPandemicsNero()])
+        bodyLbl.textAlignment = .left
+        bodyLbl.numberOfLines = 0
+        bodyLbl.lineBreakMode = .byWordWrapping
+        bodyLbl.makeSubview(of: self)
+            .width(UIScreen.main.bounds.width - 40)
+            .height(heighOfBody(text: newTxt))
+        viewMore.makeSubview(of: self)
```

Comment:
To position the viewMore button correctly, check out the code snippet below:
`    viewMore.makeSubview(of: self)
            .top(to: \.bottomAnchor, of: bodyLbl, relation: .equal, constant: -20, priority: .required)
            .trailing(to: \.trailingAnchor, of: bodyLbl, relation: .equal, constant: -10, priority: .required)
            .height(16)`

Reply:
I was looking for a way to attach the button to the end of text (which has different length), do you have any solution for that?
I mean for example if the text ends in the beginning of the line, the button appears there, if the text appear in the middle of the text the button appears there if the text ends at the end of the line the button appears there.